### DISCUSSION TITLE
Is there any example for Moments usage scenario

### DISCUSSION THREAD

**Discussion by 0xymg** [spring-projects/spring-modulith#203] [Q&A]
Hi, I look at moments repo and read readme file but could not find usage scenario or an example for that thing. Maybe I couldn't understand. Is it possible to publish an example for Moments or can you explain how to use it

**Comment by nkolosnjaji** [comment]
There is a [section](https://docs.spring.io/spring-modulith/docs/current/reference/html/#moments) referencing blog post which explains usage -> [link](https://verraes.net/2019/05/patterns-for-decoupling-distsys-passage-of-time-event/)
